src/driver/sqlite/SqliteDriver.ts

```diff
@@ -77,11 +80,55 @@ export class SqliteDriver extends AbstractSqliteDriver {
     // Protected Methods
     // -------------------------------------------------------------------------
 
+    protected createDatabaseFile(): Promise<void> {
+        return new Promise<void>((resolve, reject) => {
+            mkdirp(dirname(this.options.database), (err: NodeJS.ErrnoException) => {
```

Comment:
we cannot use nodejs dependencies because typeorm is crossplatform (there is no node deps in browser for example). replace to "any" in this case